examples/complete/fixtures.us-east-2.tfvars

```diff
@@ -1 +1,21 @@
-example = "Hello world!"
+enabled = true
+
+namespace = "eg"
+
+environment = "ue2"
+
+stage = "test"
+
+name = "msk-test"
+
+delimiter = "-"
+
+zone_id = "Z3SO0TKDDQ0RGG"
+
+availability_zones = ["us-east-2a", "us-east-2b", "us-east-2c"]
+
+kafka_version = "2.4.1"
+
+number_of_broker_nodes = 3
+
+broker_instance_type = "kafka.m5.large"
```

Comment:
If it's just a test then use smaller instance type like `kafka.t3.small` if possible

Reply:
Updated